code/frameworks/angular/template/stories/basics/component-with-provider/di.component.stories.ts

```diff
@@ -1,7 +1,8 @@
+import type { Args } from '@storybook/angular';
 import { DiComponent } from './di.component';
 
 export default {
-  title: 'Basics / Component / With Provider',
+  // title: 'Basics / Component / With Provider',
```

Comment:
the title is deactivated in order to take the one of the file ?

Reply:
Yeah, one thing we wanted to do was to rely on autotitle so it was easier to map stories back to the files they came from. I for one have often struggled to figure out which file a failing story is from due to titles. I left the names in there commented out in case someone wanted to rename the files to better match the old titles.